src/main/java/org/folio/inventory/dataimport/util/AdditionalFieldsUtil.java

```diff
@@ -582,19 +583,16 @@ private static boolean removeFirstFoundFieldByName(org.marc4j.marc.Record marcRe
   private static void replaceOclc035FieldWithNormalizedData(org.marc4j.marc.Record marcRecord,
                                               DataField dataField) {
     var variableFields = marcRecord.getVariableFields(TAG_035);
+    var index = new AtomicInteger();
```

Comment:
Are you sure we need an AtomicInteger here?

Reply:
yes, because inside of for each loop we have to use either final variable or atomic variable